src/masonite/drivers/session/RedisDriver.py

```diff
@@ -0,0 +1,147 @@
+import json
+from typing import TYPE_CHECKING
+
+from ..BaseDriver import BaseDriver
+from ...validation import MessageBag
+
+if TYPE_CHECKING:
+    from ...foundation import Application
+
+
+class RedisDriver(BaseDriver):
+    """Session driver used to store data in Redis."""
+
+    def __init__(self, application: "Application"):
+        super().__init__(application)
+        self.connection = None
+        self.driver_config = {"decode_responses": True}
+
+    def get_connection(self):
+        try:
+            import redis
+        except ImportError:
+            raise ModuleNotFoundError(
+                "Could not find the 'redis' library. Run 'pip install redis' to fix this."
+            )
+
+        if not self.connection:
+            self.connection = redis.StrictRedis(
```

Comment:
On an other note maybe we could upgrade redis version and use Redis (https://github.com/redis/redis-py#client-classes-redis-and-strictredis) since 3.X StrictRedis has been renamed to Redis.

Reply:
Yeah thats makes sense and keeps Masonite up to date with the backing module 

> On an other note maybe we could upgrade redis version and use Redis (https://github.com/redis/redis-py#client-classes-redis-and-strictredis) since 3.X StrictRedis has been renamed to Redis.

That work for me and keeps Masonite inline with the backend module